Framework/Reflectometry/test/SaveReflThreeColumnAsciiTest.h

```diff
@@ -8,19 +8,19 @@
 
 #include "MantidAPI/AlgorithmManager.h"
 #include "MantidAPI/AnalysisDataService.h"
-#include "MantidDataHandling/SaveReflThreeColumnAscii.h"
 #include "MantidDataObjects/Workspace2D.h"
 #include "MantidDataObjects/WorkspaceCreation.h"
+#include "MantidReflectometry/SaveReflThreeColumnAscii.h"
```

Comment:
You can remove this algorithm altogether, this is deprecated in favour of SaveReflectometryAscii.

Reply:
This is currently still used by the ISIS Reflectometry GUI. We'll look at removing it in #24429